Tests/Common/Util/StringAnyEncodable.swift

```diff
@@ -85,14 +87,16 @@ class StringAnyEncodableTest: UnitTest {
     }
 
     func test_stringanyencodable_encodes_complex_data() {
-        let expect = DummyData(boolean: true, numeric: 1, testValue: "foo", dict: ["test": "value"], array: [1, 2, 4])
+        let expect = DummyData(boolean: true, numeric: 1, testValue: "foo", dict: ["test": "value"], array: [1, 2, 4], hashables: ["color": ["Red", "Green", "Blue"]])
 
+        // React native wrap some values in AnyHashable
```

Comment:
Why does RN wrap values in AnyHashable? When does RN do this and when does it not?

Reply:
I think both RN and Flutter do this for all values, but not sure. More details on this can be [read above](https://github.com/customerio/customerio-ios/pull/358#discussion_r1273337734)